src/task.test.js

```diff
@@ -0,0 +1,22 @@
+import DoTask from './task.js';
+
+require('jest-localstorage-mock'); // eslint-disable-line import/no-unresolved
```

Comment:
- [ ] [OPTIONAL]  I noticed that you disabled **Eslint**. I would like to strongly recommend you remove the **eslint disable** statement and always try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: